packages/manager/modules/adp/src/deploy/deploy.html

```diff
@@ -0,0 +1,56 @@
+<section>
+    <!-- page header -->
+    <oui-page-header
+        data-heading="Deploy an Analytics Data Platform"
+        data-description="OVH Analytics Data Platform (ADP) contains all the tools to ingest, clean, process and visualize vast amounts data.
+                We deploy a preconfigured and fully secured Apache Hadoop cluster, on top of OVH Public Cloud compute and storage resources.
+                Then, simply upload your data or add data collector for real-time flows, and follow our documentation to make us of ADP benefits.">
+    </oui-page-header>
+    <!-- page content -->
+    <div class="mx-auto" style="max-width: 1200px;">
```

Comment:
Could you use Bootstrap utilities instead ? https://getbootstrap.com/docs/4.1/utilities/sizing/